### ISSUE TITLE
Sudden program termination?

### ISSUE THREAD

**Issue by ErichDonGubler** [ErichDonGubler/fzy#2]
This issue hadn't been verified yet, but I got a report of something funky happening on a ConEmu wrapper around MSYS2 (64-bit).

This was originally reported [here] (https://github.com/jhawthorn/fzy/pull/60).

**Comment by jesse23** [comment]
I can reproduce it in a clean windows 7 installation. Steps:
1. Install `msys2`.
2. Run `pacman -Syu` until all up to date.
3. Run `pacman -S fzy`.
4. Run `ls | fzy`.

Result: the prompt selection view is not pop up. And all input will be chew up after that, only output.

**Comment by jesse23** [comment]
Yes I am doing the same step but just get what you see in the former snapshot even without `conEmu`.
Never mind I've created [with](https://github.com/jesse23/with) for my requirement.

Thanks for looking at the issue.
